### DISCUSSION TITLE
What is the recommend update strategy for powerlevel10k?

### DISCUSSION THREAD

**Discussion by navinthomas** [romkatv/powerlevel10k#2626] [General]
Currently, I just follow the FAQ by executing a `git pull` at the powerlevel10k git director to update powerlevel10k. For me, it's at `${ZSH_CUSTOM:-$HOME/.oh-my-zsh/custom}/themes/powerlevel10k`. This should update the local git repo to the master repo on the remote. But, I also see various release versions at https://github.com/romkatv/powerlevel10k/releases.

Is it safer to pick the latest release tag to update powerlevel10k?

**Comment by romkatv** [comment]
It's safe to use `git pull` as per the documentation. Numbered releases aren't inherently more stable or reliable.
